File: week1/conditions.c
#include <stdio.h>
#include <cs50.h>

// A program that prints out a message based on user inputs.

int main(void)
{
    int x = get_int("X: ");
    int y = get_int("Y: ");

    if (x > y)
    {
        printf("X is greater than Y.\n");
    }
    else if (x < y)
    {
        printf("X is less than Y.\n");
    }
    else
    {
        printf("X is equal to Y.\n");
    }
}

File: week6/conditions.py
# A program that checks if one int is greater, less or equal to another

from cs50 import get_int

# Prompt user for x
x = get_int("x: ")

# Prompt user for y
y = get_int("y: ")

# Check for conditions
if x > y:
    print("x is greater than y")
elif x < y:
    print("x is less than y")
else:
    print("x is equal to y")